config/config.go

```diff
@@ -54,6 +59,22 @@ func Initialize() error {
 	} else {
 		cfg.AwsRegion = awsRegion
 	}
+
+	if metricsPort := os.Getenv(EnvMetricsPort); len(metricsPort) == 0 {
+		log.Printf("Environment variable '%s' not specified, defaulting to 8080", EnvMetricsPort)
+		cfg.MetricsPort = 8080
+	} else {
+		port, err := strconv.Atoi(metricsPort)
+		if err != nil {
+			return fmt.Errorf("invalid value for '%s': %s", EnvMetricsPort, err)
+		}
+		cfg.MetricsPort = port
+	}
+
+	if metrics := strings.ToLower(os.Getenv(EnvMetrics)); len(metrics) != 0 {
+		cfg.Metrics = true
+	}
+
```

Comment:
```suggestion
	if metricsPort := os.Getenv(EnvMetricsPort); len(metricsPort) == 0 {
		log.Printf("Environment variable '%s' not specified, defaulting to 8080", EnvMetricsPort)
		cfg.MetricsPort = 8080
	} else {
		port, err := strconv.Atoi(metricsPort)
		if err != nil {
			return fmt.Errorf("invalid value for '%s': %s", EnvMetricsPort, err)
		}
		cfg.MetricsPort = port
	}
	if metrics := strings.ToLower(os.Getenv(EnvMetrics)); len(metrics) != 0 {
		cfg.Metrics = true
	}
```
Just to clear up the empty lines